exporters/pelorus/utils.py

```diff
@@ -157,7 +157,9 @@ def get_env_var(var_name: str) -> Optional[str]:
     ...
 
 
-def get_env_var(var_name: str, default_value: Optional[str] = None) -> Optional[str]:
+def get_env_var(
+    var_name: str, default_value: Optional[str] = None, is_required: bool = False
```

Comment:
Comment to myself: Check which use case failed.

Reply:
@KevinMGranger ok the issue was as follows, but it's resolved by adding "" as default value for GIT_API, so removed changed to the utils.py:

```
05-20-2022 06:44:40 INFO No GIT_USER and no GIT_TOKEN given. This is okay for public repositories only.
Traceback (most recent call last):
File "/opt/app-root/src/committime/app.py", line 58, in <module>
git_api = pelorus.utils.get_env_var("GIT_API")
File "/opt/app-root/lib64/python3.9/site-packages/pelorus/utils.py", line 199, in get_env_var
raise ValueError(f"default value not present for SHELL env var: {var_name}")
ValueError: default value not present for SHELL env var: GIT_API
```

The more I think about it the more I am convinced that raising ValueError is the right thing to do, so we can catch issues like above during CI.

Reply:
+1 to raising ValueError. No silent failures!